amundsen_application/api/mail/v0.py

```diff
@@ -96,3 +100,11 @@ def notification() -> Response:
         recipients=data['recipients'],
         sender=data['sender']
     )
+
+
+@action_logging
+def _notification(*,
+                  notification_type: str,
+                  options: str) -> None:
```

Comment:
This should be a `Dict`. In addition, this should be bundled with `send_notification`. As is, we wouldn't catch logs for the other cases.